### PR TITLE
One more feedback option

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
This PR creates a new "Give us feedback" menu in the Navigation bar containing:
- Issue in Github
- Issue in Fogbugz (internal only)
- Anonymous feedback

image

## Related issue number
Requested by @calys
closes ITISFoundation/osparc-issues#292

## Checklist

- [ ] Did you change any service's API? Then make sure to bundle document and upgrade version (``make openapi-specs``, ``git commit ...`` and then ``make version-*``)
- [ ] Unit tests for the changes exist
- [ ] Runs in the swarm
- [ ] Documentation reflects the changes
- [ ] New module? Add your github username to [.github/CODEOWNERS](.github/CODEOWNERS)

### CODE REVIEW

**Review by pcrespov — APPROVED** [review]
If it is not too much extra effort. Can you do the same with the manuals, i.e. show them as a menu?

**Review by GitHK — APPROVED** [review]
It is not clear to me the difference between "Continue" and "Continue in ..." options.

**Review by sanderegg — APPROVED** [review]

**Comment by odeimaiz** [comment]
> If it is not too much extra effort. Can you do the same with the manuals, i.e. show them as a menu?

Done

**Comment by odeimaiz** [comment]
> It is not clear to me the difference between "Continue" and "Continue in ..." options.
> I would prefer some on hover tip explaining what is the difference.

We had something similar to what you mention and we were asked to change it to the way it is now.

File: services/web/client/source/class/osparc/component/widget/NewFogbugzIssue.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+/**
+ * Here is a little example of how to use the widget.
+ *
+ * <pre class='javascript'>
+ *   const url = osparc.component.widget.NewFogbugzIssue.getNewIssueUrl();
+ *   window.open(url);
+ * </pre>
+ */
+
+qx.Class.define("osparc.component.widget.NewFogbugzIssue", {
```
Comment (ignapas) [inline services/web/client/source/class/osparc/component/widget/NewFogbugzIssue.js:27]: this is some static utility code, should it be under component or widget... inside the widget that uses it? or inside the utils class?

Comment (odeimaiz) [reply services/web/client/source/class/osparc/component/widget/NewFogbugzIssue.js]: Done

**Pull request merged**